doc/sphinx/explanation/libraries.mmd

```diff
@@ -0,0 +1,18 @@
+flowchart
+    miral ---> mirserver
+    miral ---> mircommon
```

Comment:
Are you intentionally omitting `miral ---> mircore` to simplify the diagram?

Reply:
Nope! Just forgot it